### PR TITLE
Replace links to eth2.0-specs with consensus-specs

### PR DESCRIPTION
**Pull request by jtraglia**
This PR replaces language referring to Ethereum 2.0 with the current terminology.

### CODE REVIEW

**Review by ralexstokes — COMMENTED** [review]
nice!

im a little hesitant to touch the source code of the deposit contract files, on the basis of keeping the source the same for a very critical contract to the ethereum protocol

you are only touching the comments so its likely ok, but I could imagine consumers relying on some fingerprint of this file that would then change after this PR

**Comment by jtraglia** [comment]
> im a little hesitant to touch the source code of the deposit contract files, on the basis of keeping the source the same for a very critical contract to the ethereum protocol

Yes, that's a valid concern. I also think it should be fine, but I can revert those changes if we want.

**Comment by jtraglia** [comment]
I just realized that "eth2spec" is all over the place & wouldn't be easy to replace. We can save that for another day.

File: solidity_deposit_contract/deposit_contract.sol [hunk 1]
```diff
@@ -12,8 +12,8 @@
 pragma solidity 0.6.11;
 
 // This interface is designed to be compatible with the Vyper version.
-/// @notice This is the Ethereum 2.0 deposit contract interface.
-/// For more information see the Phase 0 specification under https://github.com/ethereum/eth2.0-specs
+/// @notice This is the Ethereum staking deposit contract interface.
+/// For more information see the Phase 0 specification under https://github.com/ethereum/consensus-specs
```
Comment (hwwhww) [inline solidity_deposit_contract/deposit_contract.sol:16]: I think we should leave it for the deposit contract so it can match the history accurately. 😄

Comment (jtraglia) [reply solidity_deposit_contract/deposit_contract.sol:16]: Yeah that makes sense! Will revert changes in that directory.

**Pull request merged**